catalog/spatial/geocoding/spatial-geocoding-offline-catalog/pom.xml

```diff
@@ -176,7 +191,11 @@
                             spatial-geocoding-api-impl,
                             spatial4j,
                             noggit,
-                            platform-util
+                            platform-util,
+                            platform-utils,
+                            commons-validator,
```

Comment:
✏️ Same comment about embedding these commons libraries. Also, I don't see `platform-utils`, does that exist?